app/assets/javascripts/questionnaire_logic.js.erb

```diff
@@ -21,7 +21,7 @@ function toggle_otherwise_field() {
 function show_hide_entry(entry, show) {
   var klass = '.' + entry + '_toggle';
   if (show) {
-    $(klass).removeClass('hidden').find('input:visible,textarea:visible').prop('disabled', false);
+    $(klass).removeClass('hidden').find('input:visible,textarea:visible,input[type=hidden]').prop('disabled', false);
```

Comment:
Omdattie anders de hidden input fields met de stop_subscription velden disabled laat als de vraag waar ze bij horen eerst gehidden was en daarna getoond werd (door bv het checken van een checkbox of radio).